core-bundle/src/Resources/config/services.yml

```diff
@@ -1,3 +1,11 @@
+parameters:
+    contao.content_element.templates: []
+    contao.frontend_module.templates: []
+    contao.content_proxy: Contao\ContentProxy
+    contao.module_proxy: Contao\ModuleProxy
```

Comment:
Not sure if we really need this. The check within the listener is there to not try to extract a template from the proxy class. However, since the proxy class is unlikely to ever have `$strTemplate` define, we could ignore it.

Reply:
Isn't the variable is defined but empty in the parent class?

Reply:
These shouldn't be parameters because they don't need to be configured.

Reply:
> Isn't the variable is defined but empty in the parent class?

Do you mean `$strTemplate` is not defined in the proxy class? Yes, I implemented this only as a fail safe as we should not ever try to retrieve the default template from the proxy class. However we can remove that check and let `getTemplateFromObject` check for the empty `$strTemplate` variable. Does not make much of a difference.